package.json

```diff
@@ -70,7 +70,6 @@
     "fix-webm-duration": "^1.0.5",
     "hast-util-find-and-replace": "^5.0.1",
     "i18next": "^21.6.14",
-    "isomorphic-ws": "^4.0.1",
```

Comment:
Why not v12? Also, shouldn't we see an update in the `yarn.lock` file?

Reply:
You do not see update in yarn lock, because the dep is installed for stream-chat and that does not change.